pre_commit_hooks/requirements_txt_fixer.py

```diff
@@ -19,8 +19,8 @@ def __lt__(self, requirement):
             return False
         else:
             return (
-                self.value.partition(b'==') <
-                requirement.value.partition(b'==')
+                self.value.lower().partition(b'==') <
```

Comment:
The assumption is made that the requirements files are all pining.  This works well enough for the others as well.